Server/ReasnAPI/ReasnAPI/Services/ImageService.cs

```diff
@@ -120,10 +126,32 @@ public bool DeleteImageById(int id)
 
         context.Images.Remove(image);
         context.SaveChanges();
-        return true;
+    }
+
+    public void DeleteImageRelatedToEvent(int id, string slug)

```

Comment:
We can't pass id since it's not in dto, only identifier would be slug. Also - don't really understand the logic here, this will (I assume) never work, since you assume that event and image have the same ids?

Reply:
it was buged i changed it idea is to check if image is releted to this event and if it is then remove it

Reply:
You still assume that image have the same id as event, that did not change, line 139

Reply:
changed